docs/source/weights_loading.rst

```diff
@@ -111,7 +111,7 @@ You can disable checkpointing by passing
 
 
 The Lightning checkpoint also saves the arguments passed into the LightningModule init
-under the `module_arguments` key in the checkpoint.
+under the `hyper_parameters` key in the checkpoint.
```

Comment:
need to call `self.save_hyperparameters()` to make this work. In the code too below.

Reply:
updated!